web-console/knikubevirt/windows-vms/windows-vms.md

```diff
@@ -0,0 +1,62 @@
+
+## Create Windows VM flow  
+
+1. In the General step, when the user chooses a Windows OS from the Operating system drop-down menu, they are notified that the Windows guest tools will be mounted as a disk, which will then show in Advanced > Virtual hardware (maybe we should label this as a CD-ROM?)
+
+![user chooses windows OS from the Operating system drop down menu](img/General1.png)
+
+![user is notified that Windows guest tools will be mounted](img/General2advanced.png)
+
+2. Storage and Networking steps
+
+In order to allow for the guest agent drivers to be installed,the user must create a NIC which uses the SATA Interface. Once the drivers are installed, the user can switch to the Virtio-scsi interface if they wish.
+
+So at first, since the interface would be pre-populated with SATA, the user will see the SATA interface (because that is the one needed to install Windows).
+
+When a Windows VM is chosen, the disk interface will not be able to be Virtio-scsi because the VM will not have the drivers for it, so we need to advise the user to install the drivers after the VM has been created and then switch the interface from SATA to Virtio-scsi (to do this switch they will need to power off and restart the VM).
+The drivers needed for Virtio-scsi are available via the Windows Guest Agent which is available by default.
+
+The user will be notified about the drivers in the Add NIC modal
+![user is notified about the drivers](img/Networking1.png)
+
+As well as in the Add disk modal
+
+![user is notified about the drivers](img/Storage1.png)
+
+3. In the list view the user sees that the VM is running but there is a warning icon indicating something is wrong. Clicking it will show a popover.
+![warning icon allowing user to know that something is going on with this VM](img/Notification1.png)
+
+4. In the status popover they are provided with an explanation they should now switch the interface to Virtio-scsi for better performance and an action link “Switch interface”. 
+They will also be notified that the VM must be restarted for the driver update to take effect.
+
+![Popover with explanation](img/NotificationPopover.png)
+
+If they choose to switch Interfaces they will be asked when they choose to schedule it
+
+![user can schedule to switch interface](img/SwitchInterface.png)
+
+This is another option that allows the user to change the interface without scheduling a specific time, so just a dropdown with the current interface selected by default (the other option in the drop down would be Virtio-scsi interface) and then the user can either Save or Cancel the change
+![user can switch interface](img/SwitchInterfaceOp2.png)
+
+The users would need to be aware that switching the interface for the disk is useful (but not required). It’s important to note that it’s not required because they will not get an error stating there is an issue.
```

Comment:
Should we be more clear about this? We hint at it in the initial popover but don't state it explicitly anywhere in these modals, WDYT?

Reply:
Do you think we should declare in the modals as well that switching the interface is useful but not mandatory?

Reply:
+1 I think it's worth exploring to make it clear